www/class/centreonGMT.class.php

```diff
@@ -213,6 +213,35 @@ public function getDate($format, $date, $gmt = null)
         return $return;
     }
 
+    /**
+     * @param $date
+     * @param null $gmt
+     * @return int|string
+     */
+    public function getCurrentTime($date, $gmt = null)
+    {
+        $return = "";
+        if (!$date) {
+            $date = "N/A";
+        }
+        if ($date == "N/A") {
+            return $date;
+        }
+
+        if (!isset($gmt)) {
+            $gmt = $this->myGMT;
+        }
+
+        if (isset($date) && isset($gmt)) {
```

Comment:
$gmt seems to be always set (set on line 232)

Reply:
change done